src/Pages/Login/index.js

```diff
@@ -1,7 +1,179 @@
 import React, { Component } from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import { fetchPost } from '../../utils/fetches';
+import { REGEXP, validate } from '../../utils/regex';
+import './index.scss';
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { userId: '', userPw: '', loggedUser: {} };
+  }
+
+  validateInputData = (id, pw) => {
+    return (
+      validate(id, REGEXP.emailRegExp) && validate(pw, REGEXP.passwordRegExp)
+    );
+  };
+
+  handleInput = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+
+    const { userId, userPw } = this.state;
+
+    if (!this.validateInputData(userId, userPw)) return;
+
+    fetchPost('http://10.58.6.148:8000/users/login', {
+      email: userId,
+      password: userPw,
+    })
+      .then((response) => response.json())
+      .then((res) => {
+        console.log('res', res);
+        if (res.token) {
+          localStorage.setItem('token', res.token);
+          console.log(res['user_name']);
+          this.setState({
+            loggedUser: res['user_name'],
+          });
+
+          console.log(res.message);
+          alert(`Login Success ${res.message}`);
+          this.props.history.push('/hotproducs');
+        }
+      })
+      .catch((error) => {
+        console.log(`error ${error.message}`);
+        alert(`error ${error.message}`);
+      });
+  };
 
-export default class Login extends Component {
   render() {
-    return <div>login</div>;
+    const { userId, userPw } = this.state;
+
+    return (
+      <>
+        <div>
+          <div className="loginPage">
+            <div className="loginContainer">
+              <div className="loginWrap">
+                <div className="loginBanner">
+                  <div className="bannerWrap">
+                    <div className="info">
+                      <p className="strongTxt">
+                        Pet shop계정 하나로 충분합니다.
+                      </p>
+                      <p className="description">
+                        Pet shop의 모든 서비스 뿐 아니라 Pelon, Paum등 다른
+                        다양한 서비스까지 <br />
+                        이제 펫샵 계정으로 이용해 보세요!
+                      </p>
+                    </div>
+                    <img
+                      alt="login banner"
+                      src="http://localhost:3000/images/banner_login.png"
```

Comment:
```suggestion
                      src="/images/banner_login.png"
```
배포되어 주소가 달라지거나 포트번호가 달라질 경우 이미지가 열리지 않을 것 같네요

Reply:
수정했습니다.